Sources/BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -174,16 +160,15 @@ public class KeyringStore: ObservableObject {
       return
     }
     Task { @MainActor in // fetch all KeyringInfo for all coin types
-      let selectedCoin = await walletService.selectedCoin()
-      let selectedAccountAddress = await keyringService.selectedAccount(selectedCoin)
+      let selectedAccount = await keyringService.allAccounts().selectedAccount
```

Comment:
should we have a fallback `selectedAccount`? even though its unlikely to happen.

Reply:
I think we should be okay, as core documentation states that `selectedAccount` will only be nil in their unit tests (in our tests using `Test*` classes we always provide a non-nil account).
https://github.com/brave/brave-core/blob/master/components/brave_wallet/common/brave_wallet.mojom#L643-L645